lms/djangoapps/courseware/views/views.py

```diff
@@ -345,6 +349,7 @@ def get_last_accessed_courseware(course, request, user):
             # TODO: (Experimental Code). See https://openedx.atlassian.net/wiki/display/RET/2.+In-course+Verification+Prompts
             'upgrade_link': check_and_get_upgrade_link(request, user, course.id),
             'upgrade_price': get_cosmetic_verified_display_price(course),
+            'course_tools': course_tools,
```

Comment:
@andy-armstrong: This TODO/ENDTODO seems like an odd construct.  Not sure if you want to get rid of it by mentioning which elements the TODO applies to in the comment (preferred), or just move the course_tools outside of this weird block?